src/login/logind.h

```diff
@@ -108,6 +108,7 @@ struct Manager {
         HandleAction handle_suspend_key_long_press;
         HandleAction handle_hibernate_key;
         HandleAction handle_hibernate_key_long_press;
+        HandleAction handle_secure_attention_key_press;
```

Comment:
drop the `_press` suffix. the other lines use the suffix `_long_press` for the long press stuff, but nothing just uses `_press`, this shouldn't either